platformsh-laravel-env.php

```diff
@@ -136,3 +135,11 @@ function mapPlatformShRedisSession(string $relationshipName, Config $config) : v
     setEnvVar('REDIS_HOST', $credentials['host']);
     setEnvVar('REDIS_PORT', $credentials['port']);
 }
+
+
+function mapPlatformShMail(string $relationshipName, Config $config) : void
+{
+    setEnvVar('MAIL_DRIVER', 'smtp');
+    setEnvVar('MAIL_HOST', $config->smtpHost);
+    setEnvVar('MAIL_PORT', '25');
```

Comment:
default is 587

Reply:
sorry, it's 25, I remembered wrong the first time

Reply:
yeah .. I mean 587 is the default in the laravel config